pennylane_lightning/lightning_qubit/_state_vector.py

```diff
@@ -49,43 +52,15 @@ class LightningStateVector:
     """
 
     def __init__(self, num_wires, dtype=np.complex128, device_name="lightning.qubit"):
-        self._num_wires = num_wires
-        self._wires = Wires(range(num_wires))
-        self._dtype = dtype
-
-        if dtype not in [np.complex64, np.complex128]:  # pragma: no cover
-            raise TypeError(f"Unsupported complex type: {dtype}")
-
         if device_name != "lightning.qubit":
             raise DeviceError(f'The device name "{device_name}" is not a valid option.')
 
-        self._device_name = device_name
-        self._qubit_state = self._state_dtype()(self._num_wires)
-
-    @property
-    def dtype(self):
-        """Returns the state vector data type."""
-        return self._dtype
-
-    @property
-    def device_name(self):
-        """Returns the state vector device name."""
-        return self._device_name
-
-    @property
-    def wires(self):
-        """All wires that can be addressed on this device"""
-        return self._wires
+        super().__init__(num_wires, dtype)
 
-    @property
-    def num_wires(self):
-        """Number of wires addressed on this device"""
-        return self._num_wires
+        self._device_name = device_name
```

Comment:
Please check if this parameter is still useful.

Reply:
I checked and It is no longer needed. :smile:

Reply:
I double-checked. However, running the test suit I discovered that `self._device_name` is needed for `QuantumScriptSerializer`